tests/src/main/java/com/hazelcast/simulator/tests/map/IntByteMapTest.java

```diff
@@ -58,6 +59,7 @@
 
     private IMap<Integer, Object> map;
     private int[] keys;
+    private byte[][] values;
```

Comment:
this means a total size of dataset can't be larger then it fits into a memory of a single member/client.